app.js

```diff
@@ -21,22 +21,22 @@ function initializeExtension() {
   }
 
   // the button
-  const newBtn = $(Handlebars.templates.button().replace(/(\r\n|\n|\r)/gm,""))
+  const newBtn = $(Handlebars.templates.button().replace(/(\r\n|\n|\r)/gm, ""))
 
   // the modal
-  const context = {confirmText: 'Are you sure you want to clone this issue to another repository? Choose whether to clone and close or clone and keep the original issue open.'}
-  const popup = $(Handlebars.templates.modal(context).replace(/(\r\n|\n|\r)/gm,""))
+  const context = { confirmText: 'Are you sure you want to clone this issue to another repository? Choose whether to clone and close or clone and keep the original issue open.' }
+  const popup = $(Handlebars.templates.modal(context).replace(/(\r\n|\n|\r)/gm, ""))
 
   // get url
   const urlObj = populateUrlMetadata()
 
   // if the page is not a pull request page(view or create)
   // and the page is not a new issue page
   // and there is no Kamino button in the DOM, proceed
-  if (urlObj.url.indexOf(urlObj.organization + '/' + urlObj.currentRepo + '/compare/') < 0 &&
-    urlObj.url.indexOf(urlObj.organization + '/' + urlObj.currentRepo + '/pull/') < 0 &&
-    urlObj.url.indexOf(urlObj.organization + '/' + urlObj.currentRepo + '/issues/new') < 0 &&
-    $('.kaminoButton').length === 0) {
+  if (urlObj.url.indexOf`(${urlObj.organization}/${urlObj.currentRepo}/compare/`) < 0 &&
+    urlObj.url.indexOf`(${urlObj.organization}/${urlObj.currentRepo}/pull/`) < 0 &&
```

Comment:
The template literals are messed up here. Should be
```suggestion
    urlObj.url.indexOf(`${urlObj.organization}/${urlObj.currentRepo}/pull/`) < 0 &&
```

same with the next line down